### ISSUE TITLE
Youtube Music is loading several minutes due to some SSL failure

### ISSUE THREAD

**Issue by ondrejholy** [tiliado/nuvolaruntime#578]
### Basic information

- Your Linux distribution and its version: Fedora 31
   

- Your desktop environment: GNOME
   

 - Nuvola Runtime version: 4.16.22 (22-g1bf660b)
   

 - The web app you use and its version: Youtube Music script 1.3.2 (2-g44bacab)
   

### Describe the bug

After I made a fresh installation and accepted terms, I saw "Youtube Music will load in a few seconds" for several minutes. After it, Youtube Music window was opened, but with "A web app will be loaded shortly...", which is probably another issue - https://github.com/tiliado/valacef/issues/17. This happens always, not only during the first start. It looks like to be caused by some SSL failures as per the debug log.

**Steps to reproduce the behavior**

1. `flatpak run eu.tiliado.NuvolaAppYoutubeMusic`
2. "Youtube Music will load in a few seconds" is shown for several minutes.

**Expected behavior**

Youtube Music window is loaded in a few seconds as announced...

**Debugging log**

Attach the debug log here: [nuvola.txt](https://github.com/tiliado/nuvolaruntime/files/4122208/nuvola.txt)

**Screenshots**

Attach screenshots here: ![Screenshot from 2020-01-28 12-22-41](https://user-images.githubusercontent.com/1051384/73259852-f81aec00-41c8-11ea-8b57-a02378e0b408.png)

**Additional context**

https://music.youtube.com works fine in my internet browsers, however, another Youtube Music player, GPMDP stopped working for me yesterday, so it might be related, see: https://github.com/MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-/issues/3787. But from the debug log, it seems to be rather some SSL issue for `https://tiliado.eu/api/funding/trial_of_machine/nuvolaplayer/96a9c41d4197f0335e5a4ac7979fb9a1f325ec6edc71bf400e3b71a3cf94f41f/`, although `openssl s_client -connect tiliado.eu:443` works fine, so I don't know.

**Comment by fenryxo** [comment]
Hello @ondrejholy. I cannot reproduce the SSL handsake issue. Could you update all Flatpak runtimes, reboot your system and try again?

```
flatpak update --system
flatpak update --user
```

**Comment by ondrejholy** [comment]
So I rebooted and updated Nuvola Runtime to 4.16.26 (26-g8b6a88f). I see the Youtube Music window almost immediately with https://github.com/tiliado/nuvolaruntime/commit/1c7ce12e9916b4b7f5bde41ce1b234cc88175514, however, I still see the "SSL handshake failed" warnings in the log.

**Comment by ondrejholy** [comment]
@fenryxo But I still see the YouTube Music window containing "A web app will be loaded shortly...", can you please confirm that this is know issue and is because of tiliado/valacef#17?

**Comment by pavel-aphonin** [comment]
I have analogical issue with Yandex Music script: the app starts and I see "A web app will be loaded shortly..." permanently.

Ubuntu 20.04.
GNOME version: 3.34.2.
Nuvola Runtime 4.16.26 (26-g8b6a88f), Nuvola Service 4.16.22 (22-g1bf660b) (I switch from stable branch to master for try resolve problem).
Yandex Music script: 1.7.0 (0-gdbf529a).

![image](https://user-images.githubusercontent.com/51195289/73821486-1d30df80-4805-11ea-98c1-b2dd38a2b890.png)
![image](https://user-images.githubusercontent.com/51195289/73821844-e14a4a00-4805-11ea-8981-f91d0ea261a7.png)

**Comment by fenryxo** [comment]
> @fenryxo But I still see the YouTube Music window containing "A web app will be loaded shortly...", can you please confirm that this is know issue and is because of [tiliado/valacef#17](https://github.com/tiliado/valacef/issues/17)?

No, tiliado/valacef#17 is unrelated.

**Comment by fenryxo** [comment]
> Nuvola Runtime 4.16.26 (26-g8b6a88f), Nuvola Service 4.16.22 (22-g1bf660b) (I switch from stable branch to master for to try resolve problem).

Please never switch to master unless you are told so. You are more likely to introduce new problems than to solve problems ;-)

**Comment by fenryxo** [comment]
The cause of the web page not loading is a SSL handshake error. It occurs in libsoup network library as well as in Chromium's network library. Not sure what is the cause though. Some users reported SSL handshake errors too but they disappeared eventually.

**Comment by pavel-aphonin** [comment]
The issue has disappeared after Ubuntu daily update.

**Comment by ondrejholy** [comment]
Hmm, it seems it is somehow related to kernel 5.3 which I am using to workaround some driver issues, because I rebooted to 5.4 today and it worked properly, but I had to reboot again back to 5.3 and it doesn't work again. So this is probably something totally unrelated to nuvola. So thanks for your time. So I am closing this issue and going to uninstall nuvola as it is unusable for me as signing on doesn't work.
